tests/scripts/all.sh

```diff
@@ -1030,6 +1030,21 @@ component_test_no_64bit_multiplication () {
     make test
 }
 
+component_build_uecc_baremetal_cmake () {
+    msg "build: uecc baremetal native, cmake"
+    scripts/config.pl baremetal
+    scripts/config.pl set MBEDTLS_USE_UECC
+    CC=gcc cmake .
+    make
+}
+
```

Comment:
So far, we only have build tests in `all.sh`, but no tests for the actual uECC based functionality. Could you add such?